src/main/java/at/hannibal2/skyhanni/features/inventory/ItemPickupLog.kt

```diff
@@ -122,7 +122,10 @@ object ItemPickupLog {
         if (!isEnabled() || !config.sack) return
 
         event.sackChanges.forEach {
-            val itemStack = (it.internalName.getItemStack())
+            var itemStack = (it.internalName.getItemStack())
+            // TODO this should not need to be done here but the whole internal name resolving needs a rework and this fixes it for now
+            if (it.internalName == "HAY_BALE".asInternalName()) itemStack = "HAY_BLOCK".asInternalName().getItemStack()
```

Comment:
internal names should be initialized as properties outside functions. also make it so it only calls .getItemStack() once

Reply:
fixy fix maybe me thinks